src/utils/time-function-util.js

```diff
@@ -0,0 +1,8 @@
+const timeFunction = (unixTimestamp) => {
```

Comment:
Somewhere else in the code I saw we are using `new Date(ts * 1000).toLocaleTimeString()` which would reduce file directory clutter. Creating a utility method for this may be an overkill.